daceml/onnx/onnx_importer.py

```diff
@@ -371,7 +383,9 @@ def __call__(
         if self.do_auto_optimize:
             self.auto_optimize()
 
-        self.sdfg(**inputs, **outputs, **params, **symbols)
+        compiled = self.compile_and_init()
```

Comment:
unrelated question: how do we cache after the SDFG is compiled, if at all? (i.e., between consecutive iterations)

Reply:
We don't cache at the moment in daceml